recaptcha_enterprise/snippets/src/main/java/recaptcha/account_defender/AccountDefenderAssessment.java

```diff
@@ -56,9 +59,16 @@ public static void main(String[] args) throws IOException, NoSuchAlgorithmExcept
     // Unique ID of the customer, such as email, customer ID, etc.
     String userIdentifier = "default" + UUID.randomUUID().toString().split("-")[0];
 
-    // Hash the unique customer ID using HMAC SHA-256.
-    MessageDigest digest = MessageDigest.getInstance("SHA-256");
-    byte[] hashBytes = digest.digest(userIdentifier.getBytes(StandardCharsets.UTF_8));
+    // Change this to a secret not shared with Google.
+    String HMAC_KEY = "SOME_INTERNAL_UNSHARED_KEY";
```

Comment:
Might this be better near the top so that users can spot it?  Or is it better where it is so it will be in the snippet?

Reply:
`static final` ?

Reply:
Leaning towards leaving it there at the snippet as it will be cohesive.
Making it final.